frontend/src/layout/navigation-3000/sidepanel/panels/SidePanelSupport.tsx

```diff
@@ -344,9 +240,18 @@ export const SidePanelSupport = (): JSX.Element => {
                             <Section title="Ask the community">
                                 <p>
                                     Questions about features, how-tos, or use cases? There are thousands of discussions
-                                    in our community forums.{' '}
-                                    <Link to="https://posthog.com/questions">Ask a question</Link>
+                                    in our community forums.
                                 </p>
+                                <LemonButton
+                                    type="primary"
+                                    fullWidth
+                                    center
+                                    to="https://posthog.com/questions"
+                                    targetBlank
+                                    className="mt-2"
+                                >
+                                    Ask a question
```

Comment:
```suggestion
                                    Ask the community
```
since you ask Max questions